Crabasite/Assets/Resources/english.json

```diff
@@ -12,15 +12,15 @@
     "startFindSilicate" :
     "Wow, the windshield is pretty badly damaged. Good thing I took the extra effort and scrubbed the security shutters' guide rails before heading out. But with the view obstructed like this there's no way of getting the ship out of the asteroid field in one piece. I have to make some kind of adhesive to seal the cracks in the windshield.",
     "firstTimeSeeingSilicateAsteroidWithUltimate" : 
-    "There seem to be large silicate deposits in these asteroids. That's exactly what I need to repair the windshield. Good thing I recently bought that engine-powered refinery module, I can use that to make the adhesive for the windshield. But first I have to collect enough silicate.",
+    "There seem to be large silicate deposits in these asteroids. That's exactly what I need to repair the windshield. Good thing I recently bought that engine-powered refinery module, I can use that to make the adhesive for the windshield. But first I have to find a way to break down the asteroids into smaller pieces that I can carry back to the ship. Maybe the parasite can help with that weird energy it absorbed.",
     "firstTimeSeeingSilicateAsteroidWithoutUltimate" : 
-    "There seem to be large silicate deposits in these asteroids. This is exactly what I need to repair the windshield. Good thing I recently bought that engine-powered refinery module, I can use that to make the adhesive for the windshield. But first I have to find a way to break down the asteroids into smaller pieces that I can carry back to the ship.",
+    "There seem to be large silicate deposits in these asteroids. That's exactly what I need to repair the windshield. Good thing I recently bought that engine-powered refinery module, I can use that to make the adhesive for the windshield. But first I have to find a way to break down the asteroids into smaller pieces that I can carry back to the ship.",
     "completedFindSilicate" :
     "This should be enough silicate. If I bring it to the engine I can use the refinery module to make the adhesive for the windshield.",
     "completedRefineSilicate" :
     "Got the adhesive. Time to head over to the windshield and seal the cracks.",
     "completedRepairWindshield" : 
-    "Ok, that should do it. Still can't believe how incredibly lucky I was to strand right next to those silicate deposits... Mental note: start saving up for one of these reinforced solid windshields with an environmental projection system.",
```

Comment:
the idea was not to refer to a kind of shield, but to something called ['environmental projection system'](https://cyberpunk.fandom.com/wiki/CrystalDome). We could add an explicit Reference to 'CrystalDome' to make the idea clearer

Reply:
`"Ok, that should do it. Still can't believe how incredibly lucky I was to strand right next to those silicate deposits. Mental note: start saving up for one of these reinforced solid windshields with an environmental projection system. I think they're called CrystalDome or something..."`

Reply:
Saw this a bit too late, 
is 5371673566c68b0e925775644842f6bcde49bab9 good, or should we use the version you just sent?